docs/output.md

```diff
@@ -380,6 +380,28 @@ Within nf-core/eager, when BAM trimming is activated alongside PMD-filtering, tr
 [DamageProfiler](https://github.com/Integrative-Transcriptomics/DamageProfiler)
 is a tool which calculates a variety of standard 'aDNA' metrics from a BAM file. The primary plots here are the misincorporation and length distribution plots. Ancient DNA undergoes depurination and hydrolysis, causing fragmentation of molecules into gradually shorter fragments, and cytosine to thymine deamination damage, that occur on the subsequent single-stranded overhangs at the ends of molecules.
 
+#### QualiMap
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `qualimap/`
+
+  - `<sample_id>/`
+    - `*.html`: in-depth report including percent coverage, depth coverage, GC content, etc. of mapped reads
```

Comment:
Also mention:
 - [ ] `genome_results.txt`
 - I guess there's no need to mention all the files created for the html report, but at least the directories should be mentioned, together with a broad description, telling users it's just the stuff that's in the html.
   - [ ] `css/` - HTML CSS styling used for the report
   - [ ] `images_qualimapReport/` - PNG version of images from the HTML report.
   - [ ] `raw_data_qualimapReport/` - The raw data used to render the HTML report as TXT files.